### DISCUSSION TITLE
macOS publishing fails but iOS Publishing passes

### DISCUSSION THREAD

**Discussion by rshillington** [codemagic-ci-cd/codemagic-docs#1975] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Build Id 64de6845484cab3f6ab4e891 fails but the same repo building for iOS works fine.  The macOS build did 2023/8/14 and nothing has changed on the workflow setup.

### Expected results

That the publish should work for macOS build, as it does with the iOS build.

### Actual results

{"tool-version":"6.315.14315","tool-path":"\/Applications\/Xcode-14.3.app\/Contents\/SharedFrameworks\/ContentDeliveryServices.framework\/Versions\/A\/Frameworks\/AppStoreService.framework","os-version":"13.3.1","product-errors":[{"message":"Asset validation failed","userInfo":{"NSUnderlyingError":"Error Domain=IrisAPI Code=-19241 \"Asset validation failed\" UserInfo={status=409, detail=The installer package includes files that are only readable by the root user. This will prevent verification of the application's code signature when your app is run. Ensure that non-root users can read the files in your app., id=dec952c3-da28-459d-b0ae-a3e8a2b3ef32, code=STATE_ERROR.VALIDATION_ERROR.90255, title=Asset validation failed, NSLocalizedFailureReason=The installer package includes files that are only readable by the root user. This will prevent verification of the application's code signature when your app is run. Ensure that non-root users can read the files in your app., NSLocalizedDescription=Asset validation failed}","NSLocalizedDescription":"Asset validation failed","iris-code":"STATE_ERROR.VALIDATION_ERROR.90255","NSLocalizedFailureReason":"The installer package includes files that are only readable by the root user. This will prevent verification of the application's code signature when your app is run. Ensure that non-root users can read the files in your app. (ID: dec952c3-da28-459d-b0ae-a3e8a2b3ef32)"},"code":90255}]}

### Build id (optional)

64de4c55ff7b7d7ed7d73e9b

**Comment by icarusdust** [comment]
Hi @rshillington please check [this thread](https://github.com/orgs/codemagic-ci-cd/discussions/1498) and let me know if it helps.

**Comment by icarusdust** [comment]
I am closing this thread due to inactivity. You can re-open or reply back in this thread in case of any further questions.
